docs/compute/monitoring-resources.md

```diff
@@ -37,7 +37,7 @@ You have sucessfully loaded slurmtools, a collection of functions
 
```

Comment:
On this page please turn the following into an admonition:

![Screenshot 2024-09-20 at 10 28 29 AM](https://github.com/user-attachments/assets/280b6ef7-aee4-482b-a077-d8f3591dc452)

Reply:
Thank you for turning this into an admonition. However, the previous bullet point format was not retained. Please fix this.